source/Ship.cpp

```diff
@@ -2702,33 +2724,33 @@ bool Ship::HasBays() const
 
 
 
-int Ship::BaysFree(bool isFighter) const
+int Ship::BaysFree(string category) const
```

Comment:
Changed to const reference. For a future update we might want to switch to numerical identifiers to get even better performance (since number comparisons are faster than string compares). But I'll keep that out of this change.